src/index.html

```diff
@@ -4,29 +4,54 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="form"
+      method="post"
+    >
+      <img
+        data-qa="big"
+        class="form__search"
+        src="images/Search.svg"
+        alt="search icon"
+      />
```

Comment:
Redundant, because you need to add icon as background for the input
```suggestion
```

Reply:
Thank you. But I haven't understood. If I implement this suggested change, 2 tests will fail.